i2c/oled_i2c/README.adoc

```diff
@@ -1,36 +1,36 @@
 = Attaching an OLED display via I2C
 
-This example code shows how to interface the Raspberry Pi Pico with an 128x32 OLED display board based on the SSD1306 display driver.
+This example code shows how to interface the Raspberry Pi Pico with an 128x32 OLED display board based on the SSD1306 display driver, datasheet https://cdn-shop.adafruit.com/datasheets/SSD1306.pdf[here].
 
 The code displays a series of tiny raspberries that scroll horizontally, in the process showing you how to initialize the display, write to the entire display, write to only a portion of the display, and configure scrolling.
 
-In contrast to the sensors in the examples library, the SSD1306 is not operated via direct reads/writes to registers. Instead, a list of versatile commands is defined (see datasheet) that allows the user to access all the capabilities of the driver. Ultimately, these commands end up writing to internal registers but the key point is that the I2C write process is now different. After sending a slave address, the data that follows can be either a command, flags to follow up a command or data to be written directly into the display's RAM. A control byte is required for each write after the slave address so that the driver knows what type of data is being sent.
+The SSD1306 is not operated via direct reads/writes to registers. Instead, a list of versatile commands is defined (see datasheet) that allows the user to access all the capabilities of the driver. Ultimately, these commands end up writing to internal registers but the key point is that the I2C write process is now different. After sending a slave address, the data that follows can be either a command, flags to follow up a command or data to be written directly into the display's RAM. A control byte is required for each write after the slave address so that the driver knows what type of data is being sent.
 
-This display is 32 pixels high by 128 pixels wide. These 32 pixels are partitioned into 4 pages, each 8 bits in height. In RAM, This looks roughly like:
+This display is 32 pixels high by 128 pixels wide. These 32 vertical pixels are partitioned into 4 pages, each 8 pixels in height. In RAM, This looks roughly like:
```

Comment:
Maybe it's worth adding a note that the SSD1306 can also drive OLED screens with other pixel-dimensions?

Reply:
There's another OLED in the box of things, it's a https://thepihut.com/collections/adafruit-lcds-displays/products/adafruit-monochrome-1-3-128x64-oled-graphic-display-stemma-qt-qwiic, I think?